.circleci/config.yml

```diff
@@ -1494,6 +1494,7 @@ workflows:
             op-service
             op-supervisor
             op-deployer
+            op-validator
```

Comment:
Confirming: Is the plan to have goreleaser integration targeted for later?

- If we plan to use this as a package eventually, versioned releases would be needed, so just calling that out to be on the same page :)

Reply:
I'll add it to goreleaser later, yeah.